src/include/parser/ddl/create_sequence.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "create_sequence_info.h"
+#include "parser/statement.h"
+
+namespace kuzu {
+namespace parser {
+
+class CreateSequence final : public Statement {
+public:
+    explicit CreateSequence(CreateSequenceInfo info)
+        : Statement{common::StatementType::CREATE_SEQUENCE}, info{std::move(info)} {}
+
+    inline const CreateSequenceInfo* getInfo() const { return &info; }
```

Comment:
No need to inline. This should be done by compiler automatically. Also I would just return a copy of info. So you also want to make `CreateSequenceInfo` EXPLICIT_COPY_DEFAULT_MOVE.